app/javascript/common/menu.js

```diff
@@ -0,0 +1,33 @@
+import http from './http';
+
+const updateUrl = '/migration/menu_section_url?section=migration&url=/migration%23';
+const activeMenuItemSelector = '[data-target="#menu-migration"] > div > ul > li[class="list-group-item active"]';
+const plansMenuItemId = 'menu_item_plans';
+const mappingsMenuItemId = 'menu_item_mappings';
+const settingsMenuItemId = 'menu_item_settings';
+const pathToMenuItem = {
```

Comment:
I wonder if we could have this path-to-menu-id mapping driven by the metadata in `app/javascript/react/config/config.js`, maybe by adding a `menuItemId` to each element in the `links` array there and doing something like `links.find(...).menuItemId` to find the matching id. That way we wouldn't have to edit code in two places if we want to change these routes.

If that's too much of a pain, I'm fine with it as-is. Just a thought.

Reply:
Yeah, that could be done too I guess.